library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-            *** Postictal-EEG-activity {suggestedTag=Brain-laterality, suggestedTag=Body-part-location, suggestedTag=Brain-centricity}
-
-        ** Episode-time-context-property <nowiki> {suggestedTag=Duration} [Additional clinically relevant features related to episodes can be scored under timing and context. If needed, episode duration can be tagged with suggested tag.]</nowiki>
-            *** Episode-consciousness {suggestedTag=Property-not-possible-to-determine}
-                **** Episode-consciousness-not-tested
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Episode-consciousness-affected
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Episode-consciousness-mildly-affected
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Episode-consciousness-not-affected
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-
-            *** Episode-awareness {suggestedTag=Property-not-possible-to-determine, suggestedTag=Property-exists,suggestedTag=Property-absence}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-
-            *** Clinical-EEG-temporal-relationship {suggestedTag=Property-not-possible-to-determine}
-                **** Clinical-start-followed-EEG <nowiki>[Clinical start, followed by EEG start by X seconds.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=numericClass, unitClass=timeUnits} </nowiki>
-                **** EEG-start-followed-clinical <nowiki>[EEG start, followed by clinical start by X seconds.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=numericClass, unitClass=timeUnits} </nowiki>
-                **** Simultaneous-start-clinical-EEG
-                **** Clinical-EEG-temporal-relationship-notes <nowiki> {requireChild} [Clinical notes to annotate the clinical-EEG temporal relationship.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass} </nowiki>
-
-            *** Episode-event-count <nowiki> {requireChild, suggestedTag=Property-not-possible-to-determine} [Number of stereotypical episodes during the recording.]</nowiki>
+            *** Myoclonic-seizure <nowiki> {suggestedTag=Duration}[Sudden, brief ( lower than 100 msec) involuntary single or multiple contraction(s) of muscles(s) or muscle groups of variable topography (axial, proximal limb, distal). Myoclonus is less regularly repetitive and less sustained than is clonus. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2; Duration tag from Beniczky ea Table 13.)]</nowiki>
+            *** Negative-myoclonic-seizure <nowiki> {suggestedTag=Duration}[Source: Beniczky ea 2017, Supplement 1; Duration tag from Beniczky ea Table 13.]</nowiki>
```

Comment:
It would be helpful to have a description.

Also Negative myoclonus is defined above --- it would be helpful to understand when the presence of a negative myoclonus indicates a seizure --- I don't follow when to use one versus the other.

Reply:
These are clinical definitions of seizure types established by ILAE, can't touch.